parcels/examples/example_mitgcm.py

```diff
@@ -0,0 +1,67 @@
+from datetime import timedelta as delta
+from os import path
+
+import numpy as np
+import xarray as xr
+from parcels import (
+    AdvectionRK4,
+    FieldSet,
+    JITParticle,
+    ParticleFile,
+    ParticleSet,
+    ScipyParticle,
+)
+
+ptype = {"scipy": ScipyParticle, "jit": JITParticle}
+
+
+def run_mitgcm_zonally_reentrant(mode):
+    """Function that shows how to load MITgcm data in a zonally periodic domain"""
+
+    data_path = path.join(path.dirname(__file__), "MITgcm_example_data/")
+    filenames = {
+        "U": data_path + "mitgcm_UV_surface_zonally_reentrant.nc",
+        "V": data_path + "mitgcm_UV_surface_zonally_reentrant.nc",
+    }
+    variables = {"U": "UVEL", "V": "VVEL"}
+    dimensions = {
+        "U": {"lon": "XG", "lat": "YG", "time": "time"},
+        "V": {"lon": "XG", "lat": "YG", "time": "time"},
+    }
+    fieldset = FieldSet.from_mitgcm(filenames, variables, dimensions, mesh="flat")
+
+    fieldset.add_periodic_halo(zonal=True)
+
+    def periodicBC(particle, fieldset, time):
+        if particle.lon < 0:
+            particle.lon += 1000000
```

Comment:
Would it be nicer to use a FieldSet constant for this value of 1000000? I.e. 
```python
fieldset.add_constant('domain_width', 1000000)
```
And then use `fieldset.domainwidth` in this kernel here?